packages/strapi-parts/src/MainNav/NavLink.js

```diff
@@ -57,8 +58,27 @@ const MainNavRow = styled(Row)`
   padding: ${({ theme }) => `${theme.spaces[2]} ${theme.spaces[3]}`};
 `;
 
-export const NavLink = ({ children, icon, ...props }) => {
+const Notification = styled(Row)`
+  //positionning when condensed
+  position: ${(props) => (props.condensed ? 'absolute' : '')};
+  transform: ${(props) => (props.condensed ? 'translate(35%, -50%)' : '')};
+  top: ${(props) => (props.condensed ? '0' : '')};
+  right: ${(props) => (props.condensed ? '0' : '')};
+
+  background: ${({ theme }) => theme.colors.primary600};
+  height: ${({ theme }) => theme.spaces[5]};
+  padding: ${({ theme }) => `0 ${theme.spaces[2]}`};
+  border-radius: ${({ theme }) => theme.spaces[10]};
+
+  ${Text} {
+    color: ${({ theme }) => theme.colors.neutral0} !important;
+    line-height: 0;
+  }
+`;
+
+export const NavLink = ({ children, icon, notifications, ...props }) => {
   const condensed = useMainNav();
+  const isNotification = notifications > 0;
```

Comment:
Notifications is a feature. The Parts components should not refer to a feature. Can you rename everything that refers to notifications into something more neutral. Best for this case would be: Badge. 
`notifications -> badgeContent`
`isNotification -> withBadge`

Reply:
Can you also add a `badgeAriaLabel` in order to give a custom aria-label to the badge. If you enable the screen reader in the Storybook, it says "Settings 2". We need to give more information about the badge.